br/pkg/lightning/errormanager/errormanager.go

```diff
@@ -87,19 +92,23 @@ const (
 	insertIntoConflictErrorData = `
 		INSERT INTO %s.` + conflictErrorTableName + `
 		(task_id, table_name, index_name, key_data, row_data, raw_key, raw_value, raw_handle, raw_row)
-		VALUES (?, ?, 'PRIMARY', ?, ?, ?, ?, raw_key, raw_value);
+		VALUES
 	`
+	// TODO: Do table_name, index_name, key_data, row_data need to escape.
+	sqlValuesConflictErrorData = "(%d,'%s','PRIMARY','%s','%s',x'%x',x'%x',raw_key,raw_value)"
```

Comment:
well keep using `?` for those then?

Reply:
If use `?`, we need to build a huge query and a long args array. I'm not sure if it will affect the performance. I will do a test later.

Reply:
I did a test. It shows there is no significant difference in performance after use `?`.  So I have updated it to use `?` instead.